src/server/api/routes/users.router.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const usersController = require('../controllers/users.controller');
+
+/**
+ * @swagger
+ * /modules:
```

Comment:
this should be /users

Reply:
I wasn't check to the swagger part. Thanks for warning me. I will fix them